eZ/Publish/Core/MVC/Symfony/Templating/Twig/Extension/CommonExtension.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Publish\Core\MVC\Symfony\Templating\Twig\Extension;
+
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFilter;
+
+/**
+ * Twig common extension for general use helpers.
+ */
+class CommonExtension extends AbstractExtension
```

Comment:
>  Twig common extension for general use helpers.

Which criteria need to be met by "general use helper"? IMHO this class should be named e.g.`DataAttributesExtension`: 

```suggestion
class DataAttributesExtension extends AbstractExtension
```

Reply:
I've chosen to place it in a new class because it didn't fit in any other defined extensions, with an aim to put more similar helpers here - helpers that are not strictly related to eZ Platform domain.
But I see your point, it would be more SOLID this way.